pkg/epinio/l10n/en-us.yaml

```diff
@@ -42,12 +42,31 @@ epinio:
     allPackages: See all packages
     getBinaries: Get binaries
   intro:
+    dashboard: Dashboard
     welcome: Welcome to Epinio
     blurb: The Application Development Engine for Kubernetes
+    description:  Epinio installs into any Kubernetes cluster to bring your application from source code to deployment and allow for Developers and Operators to work better together!
     learnMoreLink: https://epinio.io/
     learnMore: Learn more
     noNamespaces: Create a Namespace, then create your Applications
     getStarted: Get started
+    cards:
+      namespaces:
+        title: Namespaces
+        linkText: Create Namespace
+        description: Namespaces allows you to organize and manage your applications.
+        slotTitle: Resources
+      applications:
+        title: Applications
+        linkText: Deploy Application
+        description: Epinio uses Applications to transition your code, through build, to being deployed.
```

Comment:
Applies to applications and services. If these have parity with the `typeDescription` entries we should bring them in via, i think `{typeDescription.<type>}`

Reply:
The comment refers to the descriptions, rather than the labels